### ISSUE TITLE
Migrate from MySQL Doc Update WITH skip_foreign_keys

### ISSUE THREAD

**Issue by robert-s-lee** [cockroachdb/docs#4055]
Re: [Migrate from MySQL](https://www.cockroachlabs.com/docs/v2.1/migrate-from-mysql.html#skip-foreign-keys)

## Issue Description

`WITH skip_foreign_keys` should be added when an error message is received.  Possible give an example of the error message (below) to guide a user who may be cutting and pasting from the command.

## Suggested Resolution 

`pq: there is no unique constraint matching given keys for referenced table tablename`

**Comment by rolandcrosby** [comment]
Thanks for flagging, @robert-s-lee. On the [`IMPORT` page](https://www.cockroachlabs.com/docs/v2.1/import.html#object-dependencies), we have a section that explains when you might want to use the `skip_foreign_keys` option. If we added a similar note to the "Migrate" guide, would that solve what you're asking for here?

**Comment by robert-s-lee** [comment]
Took a couple of minutes of looking at the error message and then coming to the remedy.  I was thinking of having the error message so that user can pattern match and realize`skip_foreign_keys` is required would be helpful for someone who is not familiar.

**Comment by rolandcrosby** [comment]
Makes sense - @lhirata could you add a note to the MySQL and Postgres import guides? That error message isn't great either; I'll open another issue to see if we can improve it.
